src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int length;
+    private final int width;
+
+    public Rectangle(String figureProperty, String color, int length, int width) {
+        super(figureProperty, color);
```

Comment:
```suggestion
    public Rectangle(String name, String color, int length, int width) {
        super(name, color);
```
check other classes as well

Reply:
Done